### ISSUE TITLE
bigtest server fails with 'cannot link context to itself'

### ISSUE THREAD

**Issue by strobus** [thefrontside/bigtest#875]
Running the following command with BigTest 0.13.1 results in an error.

```shell
yarn bigtest server --launch=chrome
```

The error is:

```
Error: cannot link context to itself
    at t.n.link (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:5933)
    at t.call (/my-project/node_modules/@effection/events/node_modules/effection/dist/effection.js:1:9485)
    at t.n.enter (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:4576)
    at e.n.fork (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:3943)
    at e.r.advance (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:7849)
    at e.r.enter (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:7701)
    at t.call (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:8325)
    at t.call (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:7413)
    at t.n.enter (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:4576)
    at e.n.fork (/my-project/node_modules/@effection/node/node_modules/effection/dist/effection.js:1:3943)
error Command failed with exit code 1.
```

The bigtest.json file is:

```json
{
  "port": 28000,
  "launch": [
    "default"
  ],
  "app": {
    "url": "https://app.example.com" // note: redacted
  },
  "testFiles": [
    "test/**/*.test.{ts,js}"
  ]
}
```

Very cool project, thank you for all the great work!

**Comment by jnicklas** [comment]
Hi @strobus, thank you for trying out BigTest and thank you for reporting this!

I was able to replicate this issue. It seems there has been a regression when running BigTest without an app command, which really should have been caught by our test suite, I'm not sure why it wasn't. I will investigate this first thing on Monday, and hopefully we can get this fixed quickly.

**Comment by jnicklas** [comment]
Hi @strobus, we just released 0.13.2 which fixes this issue for me. Could you give it a try and see if it solves the problem for you as well?

**Comment by jnicklas** [comment]
Now that we've moved to effection v2, this issue is out of date, so I'm closing it.
